lib/barnes/reporter.rb

```diff
@@ -22,23 +14,14 @@ def report(env)
     def report_statsd(env)
       method = @statsd.respond_to?(:easy) ? :easy : :batch
       @statsd.send(method) do |statsd|
-        send_to_statsd statsd, :count, @counter_sample_rate, env[Barnes::COUNTERS], :'Rack.Server', @counter_dimensions.call(env)
-        send_to_statsd statsd, :gauge, @gauge_sample_rate, env[Barnes::GAUGES], :'Rack.Server', @gauge_dimensions.call(env)
+        send_to_statsd statsd, :count, @sample_rate, env[Barnes::COUNTERS], :'Rack.Server.All'
+        send_to_statsd statsd, :gauge, 1.0, env[Barnes::GAUGES], :'Rack.Server.All'
```

Comment:
Why 1.0?

Reply:
Sample rates for gauges in statsd have no meaning, but the method requires us to provide it. See: https://github.com/etsy/statsd/blob/master/stats.js#L269. It's probably a good thing to document.